lib/package/components/card.css

```diff
@@ -0,0 +1,97 @@
+@charset "UTF-8";
+
+.card {
+  display: flex;
+  gap: 1rem;
+  padding: 1rem;
+  background-color: var(--white);
+  box-shadow: var(--box-shadow);
+  position: relative;
+  overflow: hidden;
+}
+
+.card-vertical {
+  flex-direction: column;
+  width: 60rem;
+}
+
+.card-horizontal {
+  position: relative;
+  flex-direction: row;
```

Comment:
I think this is not required, as the `flex-direction` has value `row` by default.

Reply:
Yeah Thanks Bro